src/Symfony/Component/Clock/Test/ClockSensitiveTrait.php

```diff
@@ -17,6 +17,7 @@
 use Symfony\Component\Clock\Clock;
 use Symfony\Component\Clock\ClockInterface;
 use Symfony\Component\Clock\MockClock;
+
```

Comment:
Is it intended that this (and the following changes) partially reverts #59939?

Reply:
Could be some semi-conflicting rule when all needs to be applied to get final outcome.
Right now there is still lot of conflict between config and actually used style and I need to apply rule one by one. But it's only ~50 remaining issues to get clear state, then we will be able to apply whole config and stabilize the piece of code touched by multiple rules.